packages/flutter/lib/src/widgets/draggable_scrollable_sheet.dart

```diff
@@ -280,9 +280,6 @@ class _DraggableSheetExtent {
       context: context,
     ).dispatch(context);
   }
-
-  @override
```

Comment:
Why remove this? Maybe it wasn't helpful?

Reply:
I used it for testing at one point and accidentally committed it.  I'm not sure it's exposing the right params in the right order to be helpful for anything but the local test I was doing.